patches/tModLoader/Terraria/ModLoader/LocalizationLoader.cs

```diff
@@ -180,6 +180,23 @@ private static List<(string key, string value)> LoadTranslations(Mod mod, GameCu
 
 				string translationFileContents = streamReader.ReadToEnd();
 
+				if (LanguageManager.Instance.watchedMod == mod.Name) {
```

Comment:
It's not entirely unlikely that someone may edit files from multiple mods and be confused why it's not working. Perhaps a better idea would be to have a `Dictionary<string, string> overrides` which maps 'in mod path' to absolute on disk path, and is populated by the file watcher and cleared on reload?

Reply:
Ideally we'd use the asset system for translations, and just runtime replace the asset, but I'm not sure that it works on servers yet :| I have work to do there.